pkg/kritis/crd/securitypolicy/securitypolicy_test.go

```diff
@@ -208,3 +208,92 @@ func Test_OnlyFixesNotAvailablePassWithWhitelist(t *testing.T) {
 		t.Errorf("got unexpected violations: %v", violations)
 	}
 }
+
+func Test_BuiltProjectIDs(t *testing.T) {
+	t.Run("ISP has one buildProjjectIDs", func(t *testing.T) {
```

Comment:
`buildProjjectIDs ` -> `buildProjectIDs `

Reply:
Fixed in 2cd0c29